Marker/Classes/Marker.swift

```diff
@@ -0,0 +1,85 @@
+//
+//  Marker.swift
+//  Marker
+//
+//  Created by Htin Linn on 5/15/17.
+//  Copyright © 2017 Prolific Interactive. All rights reserved.
+//
+
+import Foundation
+
+/// Returns formatted Markdown text as an attributed string.
+///
+/// - Parameters:
+///   - markdownText: String with Markdown tags.
+///   - textStyle: Text style object containing style information.
+/// - Returns: Formatted Markdown text.
+public func attributedMarkdownString(from markdownText: String,
+                                     using textStyle: TextStyle) -> NSAttributedString {
+    guard let (parsedString, elements) = try? MarkdownParser.parse(markdownText) else {
+        return NSAttributedString(string: markdownText, textStyle: textStyle)
```

Comment:
I understand the fact that you want to keep the API concise, but at the same time it's misleading to the client using your function. As a client, reading at the function definition, I would not know that if for some reason Marker can't parse my parameters I will have a default behavior.